rd-net/RdFramework/Base/IRdBindable.cs

```diff
@@ -151,6 +151,25 @@ private static void Identify0(this IEnumerable? items, IIdentities ids, RdId id)
     #endregion
   }
 
+  public static class RdWireableEx
+  {
+    public static IScheduler? GetWireSchedulerIfBound(this IRdWireable wireable)
+    {
+      if (!wireable.IsBound)
+        return null;
+      
+      try
+      {
+        // can throw exception if wireable is not bound
+        return wireable.WireScheduler;
+      }
+      catch (Exception) when(!wireable.IsBound)
+      {
+        return null;
+      }
+    }
+  }
```

Comment:
It isn't totally clear to me which exceptions are meant to be caught by this block. Let's discuss this.

(same in .NET counterpart)

Reply:
Decided to reimplement this. At least, let's mark the "not bound"-kind exception with a separate type, and only catch this exception.

Ideally, we'd better get rid of this `catch` completely, and rearchitect this solution.